pwndbg/gdblib/nearpc.py

```diff
@@ -176,6 +176,84 @@ def nearpc(pc=None, lines=None, emulate=False, repeat=False) -> List[str]:
             first_pc = False
             should_highlight_opcodes = True
 
+        # If this instruction performs a memory access operation, we should tell
+        # the user anything we can figure out about the memory it's trying to
+        # access.
+        mem_access = ""
```

Comment:
Btw are the nearpc changes/detection needed here at all? I recall it was there to show up "UAF detected" or sth in disasm but I think we don't need this anymore bcoz:
1) it probably makes thing slower
2) it doesnt emulate code, doesnt it? and if so, it won't always be accurate
3) we now detect things dynamically via malloc/free hooks and watchpoints they set up?

Reply:
Nope, we don't really need it anymore. I left that there by accident :)

Regarding your points: (2) probably doesnt apply since, while it doesnt emulate, it only restricts the check to the current PC. Regardless, (1) and (3) absolutely do, so, yeah, better to take it out